pygmt/src/solar.py

```diff
@@ -62,8 +55,11 @@ def solar(
         Refer to https://en.wikipedia.org/wiki/Twilight for the definitions of different
         types of twilight.
     terminator_datetime : str or datetime object
-        Set the UTC date and time of the displayed terminator [Default is the current
-        UTC date and time]. It can be passed as a string or Python datetime object.
+        Set the date and time for the terminator calculation. It can be provided as a
+        string or any datetime-like object recognized by :func:`pandas.to_datetime`. The
+        time can be specified in UTC or using a UTC offset. The offset must be an
+        integer number of hours (e.g., -8 or +5); fractional hours (e.g., -8.5 or +5.5)
+        are cast to integer. [Default is the current UTC date and time].
```

Comment:
Maybe we can mention that it is the lower integer, e.g. 8.5 -> 8, not 9.

```suggestion
        Set the date and time for the terminator calculation. It can be provided as a
        string or any datetime-like object recognized by :func:`pandas.to_datetime`. The
        time can be specified in UTC or using a UTC offset. The offset must be an
        integer number of hours (e.g., -8 or +5); fractional hours (e.g., -8.5 or +5.5)
        are cast to the lower integer. [Default is the current UTC date and time].
```